message ServerInfo {
  required string ip = 1;
  required string port = 2;
  required string name = 3;
  required uint32 currentPlayer = 4;
  required uint32 maxPlayer = 5;
  required string country = 6;
}

message MasterServerResponse {
  enum PacketContent
  {
    IP = 1;
  }
  required PacketContent content = 1;
  optional ServerInfo server = 2;
  required uint32 place = 3;
}
